build-ios.sh

```diff
@@ -23,14 +23,14 @@ IOS_ARCH="x86_64"
 fi
 
 # Set Platform: device or simulator
-if [ "$2" == "device" ] || [ "$3" == "device" ]; then
+if [ "$2" == "device" ] || [ "$3" == "device" ] || [ "$4" == "device" ]; then
```

Comment:
When is "$4" be used, as there seems to be only 3 arguments - <release_type>, <arch_type>, <device_type>

Reply:
There are 4 arguments 
**clean** <release_type>, <arch_type>, <device_type>